notebooks/00_quick_start/sequential_recsys_amazondataset.ipynb

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
<p>minor typo:</p><p>sequential recomemnder systems take the sequence -&gt; sequential recommender systems take the sequence</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1010/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
I will correct it.

Reply:
For the above question, I cannot access  https://dev.azure.com/best-practices/recommenders/_build/results?buildId=18584

Reply:
> For the above question, I cannot access https://dev.azure.com/best-practices/recommenders/_build/results?buildId=18584

I made a change in the accessibility, everyone at MS should be able to access, can please try again?

Reply:
Yes I can access it now.